src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,32 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  align-items: center;
+  width: 166px;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-size: 13px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+
+  &--4 &__star:nth-last-child(-n + 2) {
+    background-image: url(/src/images/star.svg);
+  }
```

Comment:
delete it and move this style (background-image) inside stars__star because nth:child will overwrite it with active stars